cloud/awsprovider.go

```diff
@@ -561,3 +619,112 @@ func (*AWS) QuerySQL(query string) ([]byte, error) {
 	return nil, fmt.Errorf("Error getting query results : %s", *qrop.QueryExecution.Status.State)
 
 }
+
+type spotInfo struct {
+	Timestamp   string `csv:"Timestamp"`
+	UsageType   string `csv:"UsageType"`
+	Operation   string `csv:"Operation`
+	InstanceID  string `csv:"InstanceID`
+	MyBidID     string `csv:"MyBidID"`
+	MyMaxPrice  string `csv:"MyMaxPrice"`
+	MarketPrice string `csv:"MarketPrice"`
+	Charge      string `csv:"Charge"`
+	Version     string `csv:"Version"`
+}
+
+func parseSpotData(bucket string, prefix string, projectID string, region string, accessKeyID string, accessKeySecret string) (map[string]*spotInfo, error) {
+	log.Print("bucket " + bucket)
```

Comment:
Please `klog.V(2).Printf` these to allow lowering their verbosity

Reply:
Just deleted these, had left them in from testing but they aren't that helpful